### ISSUE TITLE
feat: accessibility focus

### ISSUE THREAD

**Issue by DreamsWave** [DreamsWave/waveOSPX#42]
### What to Add
Should be able to jump between elements with tab and the focus outline should be visible

### Key Steps

- [ ]
- [ ]
